Correct the description of key parameters (#6245)
MicrosoftDocs/PowerShell-Docs: reference/7.1/Microsoft.PowerShell.Security/ConvertTo-SecureString.md

## NOTES

> Some characters, such as emoticons, correspond to several code points in the string that contains them.  Such characters may cause problems and misunderstandings when used in a password&mdash;so it would be better to avoid them.

Reviewer: Edit for formatting and concision.

```suggestion
Some characters, such as emoticons, correspond to several code points in the string that contains
them. Avoid using these characters because they may cause problems and misunderstandings when used
in a password.
```
